data-reconciliation-app/README.md

```diff
@@ -0,0 +1,54 @@
+# CCF Data Reconciliation Application
+
+This is the repo for _CCF Data Reconciliation - sample_.
+
+## Overview
+
+In the last few years, regulatory legislation has enforced a unique Legal Entity Identifier (LEI) which allows all financial industry participants to standardize how they reference counterparties and clients. This law has been the impetus for all financial industry participants to clean up their reference data and adopt this new identification system. It has been inefficient and expensive for industry participants to maintain reference data. These datasets are critical for trade processing, risk management and regulatory reporting and therefore a high degree of accuracy is required. Yet the only way to have certainty regarding their accuracy is constant review and refresh against authoritative sources of all data, an enormous and costly task.
+
+Rather than collaborating with data providers and hiring human resources to keep these LEIs in sync, financial industry participants could leverage confidential compute platforms to create a consortium network and reconciliation service to clean up all their reference data. This would improve data quality in a compliant and cost-effective way through industry collaboration.
+
+## Architecture
+
+This is a work in progress overview of the current architecture and key interactions and integrations.
+
+![architecture diagram](./docs/architecture.png)
+
+## Getting Started
+
+To get started run `cd data-reconciliation-app && make test` to run the application locally.
+
+### Repo Layout
+
+```text
+📂
+├── docs                Sample application documentation
+│   └── adrs            All Architecture design records (ADR)
+│
+├── governance
+│   └── constitution    CCF network constitution files
+│   └── vote            Contains proposal voting acceptance and rejection logic
+│   └── nodes           CCF network nodes config and docker files
```

Comment:
There is something bugging me that this is not the right place, because dockerfiles only work down a folder hierarchy. (You cannot ../ in a dockerfile) - but let's see how we go and amend if required.

Reply:
Please resolve when you have read this! :-)